GUIDE.md

```diff
@@ -126,14 +127,9 @@ Next Step:
 ## Airwallex Native UI integration
 We provide native screens to facilitate the integration of payment functions.
 
-At first, add below code in your host Activity or Fragment, implement Activity#onActivityResult and handle the result.
+At first, call the initialization method of AirwallexStarter in your Application.
 ```kotlin
-    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        super.onActivityResult(requestCode, resultCode, data)
-        
-        // You must call this method on `onActivityResult`
-        AirwallexStarter.handlePaymentData(requestCode, resultCode, data)
-    }
+    AirwallexStarter.initialize(mApplication)
```

Comment:
Do we still need to do this since mApplication is already register during `Airwallex.initialize`?